### PR TITLE
#5 Add jetty-maven-plugin

### PR DESCRIPTION
**Pull request by yelzhanzh**
It seems that I can do pull requests from remote repository to remote repository only on github. When I am trying to do them in IDEA the request is always coming from my repository.

Please ignore previous pull request

### CODE REVIEW

**Comment by spyropoulosl** [comment]
If you set up your remote repository corrently in your local repopsitory then it would work. Follow this [1](https://help.github.com/articles/changing-a-remote-s-url/) but change the remote repository to be mine.

File: pom.xml [hunk 1]
```diff
@@ -168,20 +156,32 @@
...
+      <plugin>
+        <groupId>org.eclipse.jetty</groupId>
+        <artifactId>jetty-maven-plugin</artifactId>
+        <version>${org.eclipse.jetty.version}</version>
+        <configuration>
+          <webApp>
+            <contextPath>/core/</contextPath>
+          </webApp>
+          <httpConnector>
+            <port>8081</port>
```
Comment (spyropoulosl) [inline pom.xml:169]: It is advisable to use a maven filter for the port - your dev.properties might differ from mine and all of those might be different from Continuous Integration server's port

Comment (yelzhanzh) [inline pom.xml:169]: Yeah, good spot. Will change that accordingly.

Comment (yelzhanzh) [inline pom.xml:169]: For the remote repository I was referring to the mistake I have done before. Is this something wrong with the place I had pushed to? (Please say no)

**Pull request merged**